src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplCaseInvalidateTest.java

```diff
@@ -39,6 +41,12 @@ public void setup() {
   private void checkInvalidateCaseForStatus(CaseStatus status) {
     InvalidateCaseRequestDTO dto = CaseServiceFixture.createInvalidateCaseRequestDTO();
     dto.setStatus(status);
+    dto.setCaseId(UUID.fromString("77346443-64ae-422e-9b93-d5250f48a27a"));
```

Comment:
When you use the JSON fixture, this will match with CaseServiceFixture.UUID_0 , so that can be used instead of the hard-coded UUIDS here in the new tests written

Reply:
done